serverless-hello-world/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Your HTTP handling function, invoked with each request. This is an example
+ * function that echoes its input to the caller, and returns an error if
+ * the incoming request is something other than an HTTP POST or GET.
+ *
+ * @param {Context} context a context object.
+ * @param {object} context.body the request body if any
+ * @param {object} context.query the query string deserialzed as an object, if any
+ * @param {object} context.log logging object with methods for 'info', 'warn', 'error', etc.
+ * @param {object} context.headers the HTTP request headers
+ * @param {string} context.method the HTTP request method
+ * @param {string} context.httpVersion the HTTP protocol version
+ * See: https://github.com/knative-sandbox/kn-plugin-func/blob/main/docs/guides/nodejs.md#the-context-object
+ */
+function handle(context) {
+  if (context.method === "GET") {
+    return { statusCode: 200, message: "Welcome to the world of serverless" };
+  }
+  if (context.method === "POST") {
+    return { statusCode: 405, statusMessage: "Method not allowed" };
+  }
```

Comment:
Why we want to make the application to launch an error on other methods than GET/POST?

If we decide to avoid throwing errors, perhaps we can tweak the code and:
- Keep the first if that returns the 200 on the GET
- Remove the second if and return always 405.

That way we only return 200 in the GET. In any other cases, the if is not fired, so the return of 405 is what is going to happen.

Reply:
Done